pom.xml

```diff
@@ -188,6 +188,11 @@
             <version>${httpcomponents.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>4.1.2</version>
```

Comment:
Yeah, I was also confused because I tried to use `${httpcomponents.version}` first. I'll raise a PR tomorrow :).